api/v1alpha1/selfnoderemediationconfig_webhook.go

```diff
@@ -172,3 +175,10 @@ func validateToleration(toleration v1.Toleration) error {
 	}
 	return nil
 }
+
+func (r *SelfNodeRemediationConfig) validateDefault() error {
+	if defaultAnnotationVal, isDefaultAnnotationExist := r.Annotations[utils.IsDefaultConfigurationAnnotation]; isDefaultAnnotationExist && defaultAnnotationVal == "true" {
+		return nil
+	}
+	return fmt.Errorf("only single configuration is allowed")
```

Comment:
NIT: maybe be more specific?
```suggestion
	return fmt.Errorf("only single SNRConfig is allowed in the cluster")
```